admin/satellites/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Satellites
+description: Learn about satellite deployments to reduce global latency.
+---
+
+Workspace providers allow a single Coder deployment to provision and manage
+workspaces across multiple Kubernetes clusters and namespaces, including those
+located in other geographies, regions, or clouds.
+
+Satellites are Coder deployments that are used as access points to workspaces
+provisioned in a multi-region pattern. Satellites reduce latency for developers
+by acting as local proxies to the workspaces, keeping traffic from the
+developer's machine to the workspaces within the same region instead of
+requiring the traffic to cross regions back to the primary Coder deployment.
+
+Satellites, on the other hand, act as a secure reverse proxy to both Coder
+workspaces and the primary Coder deployment. Traffic meant for workspaces, such
+as web and SSH connections, are sent directly to the workspaces, while all other
+traffic is routed to the primary Coder deployment.
+
+[insert illustration here]
```

Comment:
do we have this completed?

Reply:
looking for one / working on one now

Reply:
Added one, we can def add some polish but I made something passable